std_msgs/Header header
string class_name
float32 confidence
float32 x_min
float32 y_min
float32 x_max
float32 y_max
float32 x_3d       # X position in 3D space (cm)
float32 y_3d       # Y position in 3D space (cm)
float32 z_3d       # Z position in 3D space (cm)
float32 distance   # Distance to object (cm) 